app/components/preprint-status-banner/style.scss

```diff
@@ -64,6 +64,9 @@
     .reviewer-feedback {
       padding-right: 15px;
 
+      .button-loading-indicator {
+        padding: 5px;
```

Comment:
This makes the content a little off-center, and the button is taller than the make/modify decision button:
![screen shot 2017-11-07 at 4 29 27 pm](https://user-images.githubusercontent.com/6776190/32518876-e8761b82-c3d8-11e7-8510-1b3971501b56.png)

Could you keep the horizontal space between the icon and text, without adding space vertically or to the left?